frontend/src/app/(core)/events/page.tsx

```diff
@@ -21,15 +21,15 @@ const Page = async () => {
         {events?.length ? (
           <div className='grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-8'>
             {events?.map(
-              ({ cover_image, start_date, name, venue, id, event_mode, seats_left, slug }, index) => (
```

Comment:
Can add screenshot in description for better describing what changes are done in UI

Reply:
# Before:
<img width="470" alt="Screenshot 2024-10-02 at 11 00 33 AM" src="https://github.com/user-attachments/assets/4fd41682-82d1-48a8-8e44-11138bcab561">

# After
<img width="455" alt="Screenshot 2024-10-02 at 10 59 57 AM" src="https://github.com/user-attachments/assets/3eb9919f-15e2-442e-ab4b-2ebdc8ef8202">